templates/prescription/general_edit.html

```diff
@@ -305,21 +305,65 @@
     {/literal}
     {if $GLOBALS.weno_rx_enable}
         {literal}
-        $('#drug').autocomplete({
-            {/literal}
-            source: 'library/ajax/drug_autocomplete/search.php?csrf_token_form=' + {$CSRF_TOKEN_FORM|js_url},
-            minLength: 3
-            {literal}
-        });
+        $("#drug").select2({
+        ajax: {
+            url: "library/ajax/drug_autocomplete/search.php",
+            dataType: 'json',
+            data: function(params) {
+                return {
+                  {/literal}
+                  csrf_token_form: {$CSRF_TOKEN_FORM|js_url},
```

Comment:
will change js_url to js_escape since no longer need to manually url encode it (I will do this if this PR tests well to keep things moving)